### ISSUE TITLE
Symlinked .spacemacs files to a version controlled .spacemacs file does not work on Windows

### ISSUE THREAD

**Issue by talha131** [syl20bnr/spacemacs#9997]
I am using Windows 10. I have cloned spacemacs repository inside my `%APPDATA%` folder. When I launch Emacs for the first time, it asks me usual question about preferring Evil and Helm mode. Then it creates a `.spacemacs` files in `%APPDATA%` folder. So far, so good.

Now, I want to use `.spacemacs` file that I have in my dotfiles repository. To do so, I open up cmd with administrative privileges, and use following command

    $ mklink %APPDATA%\.spacemacs %HOMEPATH%\Repos\dotfiles\spacemacs\spacemacs

It successfully creates a soft linked `.spacemacs` file. But when I launch Emacs, it asks me again about Evil and Helm mode and overwrites original file, `%HOMEPATH%\Repos\dotfiles\spacemacs\spacemacs`.

If I instead of creating a soft link, copy `spacemacs` file from repository to the folder, then Emacs respects the settings in it.

How do I make Emacs to treat this soft linked `.spacemacs` as a normal configuration?

I have similar setup on macOS where soft link poses no issues.

I am not sure it is a bug or some miss-configuration on my part.

**Comment by max-lg** [comment]
I've noticed a similar issue.
When I use cmd.exe and run: `cd %appdata%; ls -alh`
I see that the spacemacs file is named ".spacemacs" while the windows shortcut is named ".spacemacs.lnk" - even though when viewed visually with File Explorer shows up as just ".spacemacs" even when configured to show file extensions.

However, the fix for me was to use 'mklink' rather than right click + "Create shortcut" on my .spacemacs file. After using mklink, the output of ls -alh shows
lrwxrwxrwx 1 me 197121    45 Jan  4 14:39  .spacemacs -> /c/Users/me/dotfiles/spacemacs/.spacemacs
instead of showing .spacemacs.lnk

After that my .spacemacs config was loaded in Windows, although I had to modify it to add
   dotspacemacs-elpa-https nil
per this website - https://simpletutorials.com/c/2745/How%20to%20fix%20the%20Spacemacs%20443%20Connection%20Error%20on%20Windows
in order download additional packages referenced in my config.
